tests/acceptance/conftest.py

```diff
@@ -97,46 +127,97 @@ def testerchain(project, test_registry) -> TesterBlockchain:
 
 
 @pytest.fixture(scope='module')
+def stake_info(testerchain, test_registry):
+    result = test_registry.search(contract_name=STAKE_INFO)[0]
+    _stake_info = testerchain.w3.eth.contract(address=result[2], abi=result[3])
+    return _stake_info
+
+
+@pytest.fixture(scope="module")
+def ritual_token(testerchain, test_registry):
+    result = test_registry.search(contract_name=RITUAL_TOKEN)[0]
+    _ritual_token = testerchain.w3.eth.contract(address=result[2], abi=result[3])
+    return _ritual_token
+
+
+@pytest.fixture(scope="module")
 def threshold_staking(testerchain, test_registry):
     result = test_registry.search(contract_name=MOCK_STAKING_CONTRACT_NAME)[0]
-    threshold_staking = testerchain.w3.eth.contract(
-        address=result[2],
-        abi=result[3]
+    _threshold_staking = testerchain.w3.eth.contract(address=result[2], abi=result[3])
+
+    # TODO: Relocate this to pre application setup
+    taco_application_agent = ContractAgency.get_agent(
+        TACoApplicationAgent,
+        registry=test_registry,
+        provider_uri=TEST_ETH_PROVIDER_URI,
     )
-    return threshold_staking
+
+    tx = _threshold_staking.functions.setApplication(
+        taco_application_agent.contract_address
+    ).transact()
+    testerchain.wait_for_receipt(tx)
+
+    return _threshold_staking
+
+
+@pytest.fixture(scope="module", autouse=True)
+def coordinator_agent(testerchain, test_registry):
+    """Creates a coordinator agent"""
+    coordinator = ContractAgency.get_agent(
+        CoordinatorAgent, registry=test_registry, provider_uri=TEST_ETH_PROVIDER_URI
+    )
+    tx = coordinator.contract.functions.makeInitiationPublic().transact()
+    testerchain.wait_for_receipt(tx)
+    return coordinator
+
+
+@pytest.fixture(scope="module")
+def global_allow_list(testerchain, test_registry):
+    result = test_registry.search(contract_name=GLOBAL_ALLOW_LIST)[0]
+    _global_allow_list = testerchain.w3.eth.contract(address=result[2], abi=result[3])
+    return _global_allow_list
 
 
 @pytest.fixture(scope="module")
-def staking_providers(testerchain, test_registry, threshold_staking):
-    pre_application_agent = ContractAgency.get_agent(
-        PREApplicationAgent,
+def staking_providers(testerchain, test_registry, threshold_staking, stake_info):
+    taco_application_agent = ContractAgency.get_agent(
+        TACoApplicationAgent,
         registry=test_registry,
         provider_uri=TEST_ETH_PROVIDER_URI,
     )
-    blockchain = pre_application_agent.blockchain
+    blockchain = taco_application_agent.blockchain
+    minimum_stake = (
```

Comment:
In some places it called `min_authorization`, would be great to have same names everywhere

Reply:
👍 - we can address in #3227 .